terraform/projects/app-ecs-services/prometheus-service.tf

```diff
@@ -66,6 +66,14 @@ data "aws_iam_policy_document" "prometheus_policy_doc" {
   }
 }
 
+resource "aws_route53_record" "prometheus_dns" {
+  zone_id = "${data.terraform_remote_state.infra-networking-route53.dns_zone_id}"
+  name    = "metrics.${var.stack_name}.gds-reliability.engineering"
```

Comment:
This should be `prom-1.monitoring.gds-reliability.engineering` for production, and `prom-1.monitoring-staging` for staging, `prom-1.<stack name>.dev.gds-reliability.engineering` for dev stacks

See - `prometheus_subdomain` in https://github.com/alphagov/prometheus-aws-configuration-beta/blob/master/stacks/production.tfvars

Reply:
Ok this needs to be changed will look in to making this change.

Reply:
I removed this in favour of prometheus_subdomain